src/components/entity/state-badge.ts

```diff
@@ -16,12 +16,14 @@ import { HassEntity } from "home-assistant-js-websocket";
 // tslint:disable-next-line
 import { HaIcon } from "../ha-icon";
 import { HomeAssistant } from "../../types";
+import { computeActiveState } from "../../common/entity/compute_active_state";
 
 class StateBadge extends LitElement {
   public hass?: HomeAssistant;
   @property() public stateObj?: HassEntity;
   @property() public overrideIcon?: string;
   @property() public overrideImage?: string;
+  @property() public stateColor?: boolean;
```

Comment:
Should it be stateColor or colorState? 

```suggestion
  @property({type: Boolean}) public stateColor?: boolean;
```

Reply:
Is this a recent change to the syntax?

Reply:
Nope

Reply:
Lit runs in the browser when TypeScript has been transpiled away. So `@property` doesn't know the type. By specifying the type for Lit, it will help with serializing/deserializing the value to/from attributes. In the case of boolean it will become "set to true if attribute exists". 

https://github.com/Polymer/lit-element/blob/master/src/lib/updating-element.ts#L138-L164

Reply:
Got it. Thanks for the insight 👍